### PR TITLE
Prod release september 4th 24

### PR DESCRIPTION
**Pull request by michaelamar1991**

### CODE REVIEW

**Review by DeepSpace2 — APPROVED** [review]

File: templates/extender/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+      labels:
+        app: {{ include "extender.name" . }}
+        component: scheduler
+        tier: control-plane
+        version: second
+    spec:
+      serviceAccountName: {{ include "extender.name" . }}
+      containers:
+        - name: kube-{{ include "scheduler.name" . }}
+          image: public.ecr.aws/eks-distro/kubernetes/kube-scheduler:v{{(semver .Capabilities.KubeVersion.GitVersion).Major}}.{{(semver .Capabilities.KubeVersion.GitVersion).Minor}}.0-eks-{{(semver .Capabilities.KubeVersion.GitVersion).Major}}-{{(semver .Capabilities.KubeVersion.GitVersion).Minor}}-latest
```
Comment (irvingh) [inline templates/extender/deployment.yaml:27]: would suggest making this in tpl

Comment (michaelamar1991) [reply templates/extender/deployment.yaml:27]: will do it first in dev and than take it to prod

File: values.yaml [hunk 2]
```diff
@@ -58,4 +67,3 @@ storageClassName: ""
 prefixName: ""
 registry: "672188301118.dkr.ecr.eu-west-1.amazonaws.com"
```
Comment (irvingh) [inline values.yaml:68]: ?

**Review by irvingh — COMMENTED** [review]
good luck

**Pull request merged**